src/main/cpp/t04_area.cpp

```diff
@@ -32,9 +32,17 @@
 
 using namespace std;
 
-//function IsPointInArea
+bool IsPointInArea(double x, double y) {
+    bool InCircle = pow(x+1,2) + pow (y-1 , 2) <= 4;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'InCircle' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512106)